config/crds/reports/reports.kyverno.io_ephemeralreports.yaml

```diff
@@ -4,18 +4,18 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.12.0
-  name: admissionreports.reports.kyverno.io
+  name: ephemeralreports.reports.kyverno.io
 spec:
   group: reports.kyverno.io
   names:
     categories:
     - kyverno
-    kind: AdmissionReport
-    listKind: AdmissionReportList
-    plural: admissionreports
+    kind: EphemeralReport
+    listKind: EphemeralReportList
+    plural: ephemeralreports
     shortNames:
     - admr
```

Comment:
I think we need to change the names as well, should we go for 'ephr' and 'cephr'?

Reply:
good catch

Reply:
Done